src/pombot/index.js

```diff
@@ -0,0 +1,30 @@
+import {RtmClient, WebClient, MemoryDataStore} from '@slack/client';
+import {createSlackBot, createConversation, createCommand} from 'chatter';
+import config from '../../config';
+
+// define commands
+import helpCommand from './commands/help';
+
+// create bot
+const bot = createSlackBot({
+  name: 'Pombot',
+  slack: {
+    rtmClient: new RtmClient(config.tokens.pombot, {
+      dataStore: new MemoryDataStore(),
+      autoReconnect: true,
+    }),
+    webClient: new WebClient(config.tokens.pombot),
+  },
+  createMessageHandler() {
+    return createConversation([
+      this.createSlackMessageHandler({dm: true}, [
+        helpCommand
```

Comment:
In public channels, give it the name `pom`. In DM, omit the name and it will respond to all messages.